core/core.go

```diff
@@ -75,11 +81,13 @@ var (
 type Textile struct {
 	collections *c.Collections
 
-	ts    tc.NetBoostrapper
-	th    *threads.Client
-	thn   *netclient.Client
-	bucks *bc.Buckets
-	powc  *powc.Client
+	ts             tc.NetBoostrapper
+	th             *threads.Client
+	thn            *netclient.Client
+	bucks          *bc.Buckets
+	powc           *powc.Client
+	powStub        grpcdynamic.Stub
+	ffsServiceDesc *desc.ServiceDescriptor
```

Comment:
The `powStub` and `ffsServiceDesc` are dynamic gRPC representations... they are created at runtime by reading in the descriptor data from the `powergate/ffs` package that is a dependency.